sdk/messaging/azeventgrid/go.mod

```diff
@@ -0,0 +1,18 @@
+module github.com/Azure/azure-sdk-for-go/sdk/messaging/azeventgrid
+
+go 1.18
+
+require (
+	github.com/Azure/azure-sdk-for-go/sdk/azcore v1.4.0
```

Comment:
Bump to `v1.6.1` when the updated `azcore` is released on 2023-06-06.

Reply:
Done.